java/test/src/main/java/org/ray/api/test/CrossLanguageInvocationTest.java

```diff
@@ -62,12 +67,22 @@ public void testCallingPythonActor() {
     RayPyActor actor = Ray.createPyActor(PYTHON_MODULE, "Counter", "1".getBytes());
     RayObject res = Ray.callPy(actor, "increase", "1".getBytes());
     Assert.assertEquals(res.get(), "2".getBytes());
+    // Call a python function which create a python actor
+    // and pass the actor handle to callPythonActorHandle.
+    res = Ray.callPy(PYTHON_MODULE, "py_func_pass_python_actor_handle");
+    Assert.assertEquals(res.get(), "3".getBytes());
   }
 
   @Test
   public void testPythonCallJavaActor() {
     RayObject res = Ray.callPy(PYTHON_MODULE, "py_func_call_java_actor", "1".getBytes());
     Assert.assertEquals(res.get(), "Counter1".getBytes());
+    // Create a java actor, and pass actor handle to python.
+    RayActor<TestActor> actor = Ray.createActor(TestActor::new, "1".getBytes());
+    Preconditions.checkState(actor instanceof NativeRayActor);
+    byte[] actorHandleBytes = ((NativeRayActor) actor).toBytes();
+    res = Ray.callPy(PYTHON_MODULE, "py_func_call_java_actor_from_handle", actorHandleBytes);
+    Assert.assertEquals(res.get(), "12".getBytes());
```

Comment:
this can be a separate test case as well.

Reply:
OK